009_paper_GR2020/pygmt_jn_fig_2/GR_2020_Fig_2.ipynb

```diff
@@ -574,7 +574,7 @@
     "                  FONT_LABEL = '6',\n",
     "                  MAP_FRAME_PEN = '0.4p'):\n",
     "\n",
-    "    fig.legend(position = 'JBR+jBR+o0.1c', box = '+plinen+glinen+gwhite+p0.5p,black')\n",
+    "    fig.legend(position = 'JBR+jBR+o0.1c', box = '+gwhite+p0.5p,black')\n",
```

Comment:
Please indicate if you prefer other fill and pen colors (and a semi-transparency for the fill).